docs/user_guide/docker.rst

```diff
@@ -1,15 +1,66 @@
 
-How to run traffic in a Docker container?
+How to use traffic in a Docker container?
 =========================================
 
-Run a Jupyter environment with traffic (latest)
------------------------------------------------
+If you don't want to be bothered with Python's dependencies and just want to
+work, using a Docker container might be an easy solution. It is quite simple to
+run the traffic library with everything it needs in a Docker container and you
+have a few options. We provide a self contained container image on GitHub that
+you can use. It has everything in it that you need. Or, alternatively, you can
+build your own image. We describe both below.
 
-It is quite simple to run the traffic library in a Docker container. The
-following two examples show how you can use the traffic library in a docker with
-`jupyter notebook <https://jupyter-docker-stacks.readthedocs.io/en/latest/>`__.
-If you are completely unfamiliar with Docker and how to modify a Docker image,
-you can find a good tutorial in the `official documentation
+Use the GitHub container image
+------------------------------
+
+Using the container image from GitHub is simple. Just make sure you have Docker
+installed and running, then pull the image with
+
+.. code:: bash
+
+    docker pull ghcr.io/xoolive/traffic/jupyter:latest
+
+Once the image is downloaded, you can run it with
+
+. code:: bash
+
+    docker run -it -p 8888:8888 ghcr.io/xoolive/traffic/jupyter
+
+This will start the container and you can connect to it with the browser and
+work in a Jupyter notebook. Just copy-paste the URL that starts with
+*http://127.0.0.1:8888...* into your favourite browser and have fun. 
+
+Use the GitHub container image with Visual Studio Code
```

Comment:
@rmonstein I am not sure this section is necessary now. I would consider removing it.

Reply:
Actually that part was confusing for me in the first place...

Reply:
Do you mind if I divide the documentation into two parts (two seperate pages)?
One part would be how to work **with** the traffic library in a container and would be aimed at users of the library. I would more or less keep the first part of the documentation as it is since I think it shows how one can work with a container in different ways (among them just use it as a jupyter server and working in the browser).
The second part would be how to work **on** traffic and would be aimed at developers of traffic. I would expand that part a bit.
I think that would make the documentaion more clear. I agree with you, it's not well distingished at the moment.
What do you think?

Reply:
Yes good idea!!!